backend/dataall/modules/worksheets/aws/bedrock_prompts.py

```diff
@@ -0,0 +1,118 @@
+SQL_EXAMPLES = [
+    {
+        'User': """I want to get the average area of all listings \\n\\nBased on on the following glue metadata: \n            
+    <context>
+    Database Name : dataall_homes_11p3uu8f
+    Table name: listings 
+                Column Metadata: [{'Name': 'price', 'Type': 'bigint'}, {'Name': 'area', 'Type': 'bigint'}, {'Name': 'bedrooms', 'Type': 'bigint'}, {'Name': 'bathrooms', 'Type': 'bigint'}, {'Name': 'stories', 'Type': 'bigint'}, {'Name': 'mainroad', 'Type': 'string'}, {'Name': 'guestroom', 'Type': 'string'}, {'Name': 'basement', 'Type': 'string'}, {'Name': 'hotwaterheating', 'Type': 'string'}, {'Name': 'airconditioning', 'Type': 'string'}, {'Name': 'parking', 'Type': 'bigint'}, {'Name': 'prefarea', 'Type': 'string'}, {'Name': 'furnishingstatus', 'Type': 'string'}, {'Name': 'passengerid', 'Type': 'bigint'}, {'Name': 'survived', 'Type': 'bigint'}, {'Name': 'pclass', 'Type': 'bigint'}, {'Name': 'name', 'Type': 'string'}, {'Name': 'sex', 'Type': 'string'}, {'Name': 'age', 'Type': 'double'}, {'Name': 'sibsp', 'Type': 'bigint'}, {'Name': 'parch', 'Type': 'bigint'}, {'Name': 'ticket', 'Type': 'string'}, {'Name': 'fare', 'Type': 'double'}, {'Name': 'cabin', 'Type': 'string'}, {'Name': 'embarked', 'Type': 'string'}]
+                Partition Metadata: []           
+    </context>""",
+        'AI': """SELECT AVG(CAST(area AS DOUBLE))
+FROM dataall_homes_11p3uu8f.listings
+WHERE area IS NOT NULL;""",
+    },
+    {
+        'User': """I want to get the average of the 3 most expensive listings with less than 3 bedrooms\\n\\nBased on on the following glue metadata: \n            
+    <context>
+    Database Name : dataall_homes_11p3uu8f
+    Table name: listings 
+                Column Metadata: [{'Name': 'price', 'Type': 'bigint'}, {'Name': 'area', 'Type': 'bigint'}, {'Name': 'bedrooms', 'Type': 'bigint'}, {'Name': 'bathrooms', 'Type': 'bigint'}, {'Name': 'stories', 'Type': 'bigint'}, {'Name': 'mainroad', 'Type': 'string'}, {'Name': 'guestroom', 'Type': 'string'}, {'Name': 'basement', 'Type': 'string'}, {'Name': 'hotwaterheating', 'Type': 'string'}, {'Name': 'airconditioning', 'Type': 'string'}, {'Name': 'parking', 'Type': 'bigint'}, {'Name': 'prefarea', 'Type': 'string'}, {'Name': 'furnishingstatus', 'Type': 'string'}, {'Name': 'passengerid', 'Type': 'bigint'}, {'Name': 'survived', 'Type': 'bigint'}, {'Name': 'pclass', 'Type': 'bigint'}, {'Name': 'name', 'Type': 'string'}, {'Name': 'sex', 'Type': 'string'}, {'Name': 'age', 'Type': 'double'}, {'Name': 'sibsp', 'Type': 'bigint'}, {'Name': 'parch', 'Type': 'bigint'}, {'Name': 'ticket', 'Type': 'string'}, {'Name': 'fare', 'Type': 'double'}, {'Name': 'cabin', 'Type': 'string'}, {'Name': 'embarked', 'Type': 'string'}]
+                Partition Metadata: []
+    <context/>""",
+        'AI': """SELECT AVG(price) AS average_price
+FROM (
+  SELECT price
+  FROM dataall_homes_11p3uu8f.listings
+  WHERE bedrooms > 3
+  ORDER BY price DESC
+  LIMIT 3)""",
+    },
+    {
```

Comment:
nit: the second and third examples are pretty similar, maybe we can remove one of them to reduce the context sent to the LLM

Reply:
agreed - removing 3rd example